src/registers/esr_el3.rs

```diff
@@ -0,0 +1,130 @@
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+//
+// Copyright (c) 2018-2023 by the author(s)
+//
+// Author(s):
+//   - Andre Richter <andre.o.richter@gmail.com>
+//   - Berkus Decker <berkus+github@metta.systems>
+//   - Bradley Landherr <landhb@users.noreply.github.com>
+//   - rmsyn <rmsynchls@gmail.com>
+
+//! Exception Syndrome Register - EL3
+//!
+//! Holds syndrome information for an exception taken to EL3.
+
+use tock_registers::{interfaces::Readable, register_bitfields};
+
+register_bitfields! {u64,
+    pub ESR_EL3 [
+
+        /// Reserved
+        RES0 OFFSET(37) NUMBITS(27) [],
+
+        /// Instruction Specific Syndrom 2
+        ///
+        /// When FEAT_LS64 is implemented:
+        ///
+        /// When FEAT_LS64_V is implemented, if a memory access generated by an ST64BV instruction
+        /// generates a Data Abort exception for a Translation fault, Access flag fault, or Permission fault, then
+        /// this field holds register specifier, Xs.
+        ///
+        /// When FEAT_LS64_ACCDATA is implemented, if a memory access generated by an ST64BV0
+        /// instruction generates a Data Abort exception for a Translation fault, Access flag fault, or Permission
+        /// fault, then this field holds register specifier, Xs.
+        ///
+        /// Otherwise, this field is RES 0.
+        ISS2 OFFSET(36) NUMBITS(5) [],
```

Comment:
Offset should be 32

![Screenshot 2023-07-18 at 4 13 21 PM](https://github.com/rust-embedded/aarch64-cpu/assets/52972368/3a8386ba-0c35-482c-b362-d315921e9b7a)